### PR TITLE
Placeholder for MTU change (in separate branch)

### PR DESCRIPTION
**Pull request by ktbyers**
Contains Junos and IOS change.

### CODE REVIEW

**Comment by ktbyers** [comment]
IOS change from @afourmy https://github.com/napalm-automation/napalm/pull/529 was integrated in.

**Comment by ktbyers** [comment]
The IOS part of this change is good.

Junos part needs reviewed.

Still need nxos, nxos_ssh, eos, and iosxr.

**Comment by dbarrosop** [comment]
#539 contains the nxos one

**Comment by dbarrosop** [comment]
We should probably use feature branches for things like this so everybody can work independently and only merge the feature branch into develop when it's fully implemented.

**Comment by afourmy** [comment]
@dbarrosop There is a branch called "get_interface_mtu3" (see https://github.com/napalm-automation/napalm/pull/529). That's where I pushed https://github.com/napalm-automation/napalm/pull/539

**Comment by dbarrosop** [comment]
Ah, my bad, thanks for the clarification :)

**Comment by mirceaulinic** [comment]
FYI, I only resolved the conflict here. We might need to update the mock data, due to recent changes in the output structure of `get_interfaces` (which contains the logical interfaces too): https://github.com/napalm-automation/napalm/pull/518

**Comment by mirceaulinic** [comment]
Just checking in: besides IOS-XR and NXOS is there anything else to be sorted here?

**Comment by ktbyers** [comment]
Someone probably needs to own coordinating the whole task i.e. making sure all the code is good, that we have proper unit tests for it, finishing any missing drivers, and releasing it.

File: napalm/junos/junos.py [hunk 1]
```diff
@@ -293,9 +293,12 @@ def _convert_to_dict(interfaces):
                         napalm.base.helpers.mac,
                         interfaces[iface]['mac_address'],
                         py23_compat.text_type(interfaces[iface]['mac_address'])),
-                    'speed': -1
+                    'speed': -1,
+                    'mtu': -1
                 }
                 # result[iface]['last_flapped'] = float(result[iface]['last_flapped'])
+            if interfaces[iface]['mtu'] != 'Unlimited':
```
Comment (dbarrosop) [inline napalm/junos/junos.py]: I think this is missing indentation as it seems to be outside the previous for loop.

**Comment by ktbyers** [comment]
Closing...don't think it makes sense to keep this open any longer.

**Pull request closed without merging**